Documentation/subcommands/run.md

```diff
@@ -80,11 +80,13 @@ To inherit all environment variables from the parent use the `--inherit-env` fla
 
 To explicitly set individual environment variables use the `--set-env` flag.
 
+To explicitly set environment variables from a file use the `--set-env-file` flag. Variables are expected to be in the OS format (e.g. VAR_NAME=VALUE) and lines starting with `#` or `;` will be ignored.
```

Comment:
Instead of "OS format", you could say format similar to systemd's [EnvironmentFile=](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#EnvironmentFile=), and one variable per line. (to differentiate with `/proc/$pid/environ` which is separated by '\0')